src/librbd/api/Image.cc

```diff
@@ -834,6 +837,93 @@ int Image<I>::remove(IoCtx& io_ctx, const std::string &image_name,
   return cond.wait();
 }
 
+template <typename I>
+int Image<I>::flatten_children(I *ictx, const char* snap_name,
+                                  ProgressContext& pctx) {
```

Comment:
Nit: fix indentation:

```suggestion
                               ProgressContext& pctx) {
```